tests/python_tests/test_gguf_reader.py

```diff
@@ -61,15 +61,24 @@ def test_pipelines_with_gguf_generate(pipeline_type, model_ids):
 @pytest.mark.parametrize("pipeline_type", get_gguf_pipeline_types())
 @pytest.mark.parametrize("model_ids", get_gguf_model_list())
 @pytest.mark.parametrize("enable_save_ov_model", [False, True])
+@pytest.mark.parametrize("prompt", [
+    'Why is the Sun yellow?', 
+    # To check that special tokens are handled correctly.
+    '<|endoftext|> <|im_end|>', 
+    '<|endoftext|><|endoftext|><|im_end|>', 
+    '<|endoftext|> Why the Sky is Blue? <|im_end|>',
+])
 @pytest.mark.precommit
-def test_full_gguf_pipeline(pipeline_type, model_ids, enable_save_ov_model):
+def test_full_gguf_pipeline(pipeline_type, model_ids, enable_save_ov_model, prompt):
     if sys.platform == 'darwin':
         pytest.skip(reason="168882: Sporadic segmentation fault failure on MacOS.")
     gguf_model_id = model_ids["gguf_model_id"]
     gguf_filename = model_ids["gguf_filename"]
     dynamic_quantization_group_size = model_ids["dynamic_quantization_group_size"]
-    prompt = 'Why is the Sun yellow?'
 
+    if "sammysun0711" in gguf_model_id and "<|endoftext|>" in prompt:
```

Comment:
```suggestion
    if gguf_model_id == "sammysun0711/tiny-random-deepseek-distill-qwen-gguf" and "<|endoftext|>" in prompt:
```
It is better to skip the test by the model, not by the HF username.

Reply:
done